blog.html

```diff
@@ -4,12 +4,310 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Blog</title>
 </head>
 
-<body>
+<body class="pade-body">
+    <header class="page-header">
+        <a class="main-logo"><img src="img/logo.svg" alt="binabox-logo">
+            <span class="logo-text">BINABOX</span></a>
+        <nav class="top-nav">
+            <ul class="nav-list">
+                <li class="nav-item"><a href="index.html">HOME</a>
+                </li>
+                <li class="nav-item"><a href="about.html">ABOUT</a>
+                </li>
+                <li class="nav-item"><a href="roadmap.html">ROADMAP</a>
+                </li>
+                <li class="nav-item"><a href="collection.html">COLLECTION</a>
+                </li>
+                <li class="nav-item"><a href="faqs.html">FAQS</a>
+                </li>
+                <li class="nav-item"><a href="pages.html">PAGES</a>
+            </ul>
+            <div class="nav-buttons">
+                <button class="toggle-button" onclick="toggle()">
+                    <img id="sun-icon" src="light-mode-icon.png" alt="Toggle Light/Dark Mode">
+                </button>
+                <button type="button" class="discord-button">
+                    <img src="#" alt="discord-logo"> DISCORD
+                </button>
+                <button type="button" class="connect-button">
+                    <img src="#" alt="connect-logo"> CONNECT
+                </button>
+            </div>
+        </nav>
+    </header>
+    <main class="blog-main">
+        <section class="breadcrumbs-container">
+            <h1 class="main-heading">CONTACT US</h1>
+            <div class="breadcrumbs">
+                <nav class="breadcrumbs-nav">
+                    <!-- I understand this separator {&#10072;} can be styled with css instead of symbol -->
+                    <a href="index.html" class="breadcrumb-link">HOME</a> &#10072;
+                    <a href="our-blog.html" class="breadcrumb-link">OUR BLOG</a>
+                </nav>
+            </div>
+            <img class="breadcrumb-image" src="#" alt="darkgrey background">
+        </section>
+        <section class="blog-content">
+            <ul class="blog-list">
+                <li class="list-item">
+                    <figure class="item-wrap">

```

Comment:
Use <article> instead of <figure> where the content is self-contained and could be independently distributed or reused. [Learn more about the <article> element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article).